spyder/plugins/updatemanager/widgets/update.py

```diff
@@ -474,6 +519,49 @@ def start_install(self):
 
         subprocess.Popen(' '.join(cmd), shell=True)
 
+    def _start_updater(self):
+        """Start updater application."""
+        info = {
+            "install_file": self.installer_path,
+            "conda_exec": find_conda(),
+            "env_path": sys.prefix,
+            "update_type": self.asset_info["update_type"],
+            "window_title": _("Spyder update"),
+            "scale_factor": float(os.getenv("QT_SCALE_FACTOR") or 1),
+            "initial_message": _(
+                "Updating Spyder, this will take a few minutes ..."
+            ),
+            "success_message": _(
+                "The update was succesful!<br>Spyder will be launched shortly"
+            ),
+            "failure_message": _("Unfortunately the update failed"),
+            "error_message": _("There was an error in the update process"),
+            "details_title": _("Show details"),
+            "font_family": self.get_conf(
+                "app_font/family", section="appearance"
+            ),
+            "font_size": int(
+                self.get_conf("app_font/size", section="appearance")
+            ),
+            "monospace_font_family": self.get_conf(
+                "monospace_app_font/family", section="appearance"
+            ),
+            "monospace_font_size": int(
+                self.get_conf("monospace_app_font/size", section="appearance")
+            ),
+            "interface_theme": self.get_conf("ui_theme", section="appearance"),
+            "icon_color": "#FAFAFA",
```

Comment:
```suggestion
            "icon_color": SpyderPalette.ICON_1,
```

This is the right color for icons according to the theme that is being used.